README.md

```diff
@@ -1,2 +1,134 @@
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠️ Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖  MOVIE GALLERY <a name="about-project"></a>
+
+>This is my second microverse project I have built it with HTML and CSS.
+
+## 🛠️ Built With <a name="built-with">
+### Tech Stack <a name="tech-stack"></a>
+- HTML
+- JAVASCRIPT
+
+</a>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+-  User can Like a movie
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- browser like google chrome
+
+### Setup
+
+Clone this repository to your desired folder:
+
+- clone the project using by running the following command :
+
+`git clone`
+`https://github.com/hagan199/JavascriptCapstone.git`
+
+###  Install
+
+ ```install
+ npm install
+ ```
+
+### Test
+
+```test
+npx hint .
+npx eslint .
+```
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Emmanuel Hagan**
+
+- GitHub: [hagan199](https://github.com/hagan199)
+- LinkedIn: [Emmanunel Hagan](https://www.linkedin.com/in/emmanuel-hagan-26219a95/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p> 
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- **Add animation for the project**
+- **Add the editing functionality**
+- **Add the test for the project**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/hagan199/JavascriptCapstone/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐ Show your support <a name="support"></a>
+
+If you like this project you should give me a star.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for the opportunity
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://github.com/hagan199/JavascriptCapstone/issues).
+/tree/develop/MIT.md) licensed.
```

Comment:
- [OPTIONAL] This wasn't pointed out by the previous reviewer so I am going to make this optional but I strongly suggest you fix this before merging.
- - The MIT LINK in your readme is not correctly linked, Currently, when you click on it, it takes you to the issue page instead of the License file. I'll suggest you fix this as readme with correct working links makes your repository look professional :+1: :100: :smile: